### PR TITLE
Notebook Getting Started Series

### PR DESCRIPTION
**Pull request by JakeRadMSFT**
This is a getting started with Notebook series - It's just meant to gently introduce how to use Notebooks.

### CODE REVIEW

File: notebook-reference/01-What are Notebooks.ipynb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+          "execution_count": 1,
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "To add more cells you can use the buttons that show up when you hover above or below a cell ![Play Button](https://github.com/JakeRadMSFT/csharp-notebooks/raw/main/notebook-reference/images/hover%20buttons.png)  or keyboard shortcuts [Keyboard Shortcut Reference](https://github.com/JakeRadMSFT/csharp-notebooks/blob/main/notebook-reference/XX-VS%20Keyboard%20Shortcuts.ipynb)"
```
Comment (beccamc) [inline notebook-reference/01-What are Notebooks.ipynb:43]: Nit, I would make the sentence about keyboard shortcuts on a new line. The or is getting lost for me. 

image

I'd add a line break and then say "Keyboard shortcuts are also available."

File: notebook-reference/02-Code Cells.ipynb [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Built in \"Display\" function\n",
+        "The display function will give you more control over what is displayed and give you the ability to update it."
```
Comment (beccamc) [inline notebook-reference/02-Code Cells.ipynb:168]: I didn't understand what you meant by this when I read it. Maybe adding something about when it's displayed?

File: notebook-reference/03-Markdown Cells.ipynb [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Emphasis\n",
+        "This is a **bold text**.  \n",
+        "This is *Italicized text*.  \n",
+        "This is ~~strikethrough text~~.  \n",
+        "This is ==highlighted text==.  \n",
```
Comment (beccamc) [inline notebook-reference/03-Markdown Cells.ipynb:57]: It seems like 57-59 don't work?

File: notebook-reference/02-Code Cells.ipynb [hunk 4]
```diff
@@ -0,0 +1,216 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Notebook Code Cells\n",
+        "\n",
+        "This notebooks is primarily going to cover scenarios that are different in notebook - like updating output, referencing NuGets, etc...  \n",
```
[the remark this answers is not in the document]
Comment (JakeRadMSFT) [reply notebook-reference/02-Code Cells.ipynb:9]: @luisquintanilla how do you do that?

File: notebook-reference/03-Markdown Cells.ipynb (hunk 3 above)
Comment (JakeRadMSFT) [reply notebook-reference/03-Markdown Cells.ipynb:57]: Didn't work on GitHub :) - They work in VS.

**Review by beccamc — APPROVED** [review]

**Pull request merged**